pkg/gardenlet/operation/botanist/etcd.go

```diff
@@ -42,11 +40,6 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 		replicas = ptr.To(getEtcdReplicas(b.Shoot.GetInfo()))
 	}
 
-	hvpaEnabled := features.DefaultFeatureGate.Enabled(features.HVPA)
```

Comment:
The following It name can be adapted now: https://github.com/gardener/gardener/blob/008c4b4cfd1e8a14bff4cdfb4bb8e44a0d4ebe3b/pkg/gardenlet/operation/botanist/etcd_test.go#L147